Dockerfile

```
@@ -113,17 +113,18 @@ CMD python manage.py runserver 0.0.0.0:8000
 # ---- Production image ----
 FROM core as production
 
+ARG DOCKER_USER
+ARG SITE
 ARG STATIC_ROOT
 
 # Copy collected symlinks to static files
 COPY --from=collector ${STATIC_ROOT}/staticfiles.json ${STATIC_ROOT}/
 
 # Un-privileged user running the application
-ARG DOCKER_USER
 USER ${DOCKER_USER}
 
 # The default command runs gunicorn WSGI server in the sandbox
-CMD gunicorn -c /usr/local/etc/gunicorn/app.py app.wsgi:application
+CMD gunicorn -c /usr/local/etc/gunicorn/app.py ${SITE}.wsgi:application
```

For the `CMD` statement, the `${SITE}` variable will be considered as an environment variable and will be interpolated by the shell, so you have to define an environment variable from the corresponding `ARG`:

```suggestion
ENV SITE=${SITE}
CMD gunicorn -c /usr/local/etc/gunicorn/app.py ${SITE}.wsgi:application
```